content/authorization/guides/end-user/system-user/delete-system-user/_index.nb.md

```
@@ -7,11 +7,11 @@ weight: 3
 
 # Sletting av systembruker
 
-En systembruker kan kun slettes av en sluttbrukersystemleverandør. Ved sletting fjernes alle tilknyttede delegeringer automatisk. Per i dag er sletting av systembruker kun mulig via Altinn-portalen. Brukeren med rollen tilgangsstyrer i sluttbrukervirksomheten må logge inn i Altinn-portalen for å gjennomføre slettingen. Leverandør kan ikke slette systembruker på vegne av kunden.
+Det er kun sluttbrukeren selv (med rollen tilgangsstyrer) som kan slette systembrukeren. Ved sletting fjernes alle tilknyttede delegeringer automatisk. Per i dag er sletting av systembruker kun mulig via Altinn-portalen. Brukeren med rollen tilgangsstyrer i sluttbrukervirksomheten må logge inn i Altinn-portalen for å gjennomføre slettingen. 
 
 ## Slik sletter du systembruker
 
-1. Logg inn til Altinn portalen og åpne oversikten over systemtilganger: https://am.ui.tt02.altinn.no/accessmanagement/ui/systemuser/overview
+1. Logg inn til Altinn portalen og åpne oversikten over systemtilganger: https://am.ui.altinn.no/accessmanagement/ui/systemuser/overview
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)